src/app/modules/lti/pages/lti/lti.component.ts

```diff
@@ -40,7 +46,11 @@ export class LTIComponent implements OnDestroy {
   private loginId$ = this.activatedRoute.queryParamMap.pipe(map(queryParams => queryParams.get(loginIdParam)));
 
   private isRedirection$ = this.activatedRoute.queryParamMap.pipe(
-    map(queryParams => queryParams.has(isRedirectionParam)),
+    map(queryParams => queryParams.get(isRedirectionParam) === '1'),
+  );
+
+  private fromPath$ = this.activatedRoute.queryParamMap.pipe(
+    map(queryParams => (queryParams.get(useFromPathKey) === '1' ? queryParams.get(fromPathKey) : null)),
```

Comment:
I don't feel there was real need for a observable here (versus `activatedRoute.snapshot`) (as it is an immediate process and the arg is not supposed to change I assume), but that's ok.